GNUmakefile

```diff
@@ -23,7 +23,7 @@ bin/terraform:
 	(cd $(CURDIR)/bin/ ; unzip terraform.zip)
 
 testacc: fmtcheck bin/terraform
-	PATH="$(CURDIR)/bin:${PATH}" TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout=90s
+	PATH="$(CURDIR)/bin:${PATH}" TF_ACC=1 TF_LOG=DEBUG go test $(TEST) -v $(TESTARGS) -timeout=90s
```

Comment:
Could you please remove this change?
It may be beneficial, but it fills test output with thousands of lines of TF debug that will be never read.

Reply:
Why was this resolved?

Reply:
Sorry, just haven't pushed changes.